doc-390 (#961)
spectrocloud/librarium: content/docs/04.5-devx/07-sandbox-clusters.md

2. Select the `Sandbox Clusters` from the left ribbon menu, click `+ Sandbox Clusters,` and provide the following information to the app creation wizard.
   * Select the Cluster Group: From the available host cluster group, select the cluster group to host the new sandbox cluster.
   * Sandbox cluster name: Provide a custom sandbox cluster name or go with the default name.
   * Provide the resource Limits in terms of CPU, Memory, and Storage. 

 <br />

 |**Sandbox Cluster Resource ** | **Default    **   |**Minimum Limit**|
 |------------------------------|-------------------|-----------------|
 | CPU (per request)            | 6                 | 4               |
 | Memory (per request)         | 8 GiB             | 4 GiB           |

>  | Storage (per request)        | 10 GiB            | 2 GiB           |

Reviewer: The minimum for launching sandbox clusters is 2.5 CPU, 2.5 GiB memory, 0 GiB storage.
Also see https://spectrocloud.atlassian.net/browse/PPD-618

<br />

>    * Review the information and deploy the sandbox cluster. The sandbox cluster will get provisioned and start running in a few minutes.

Reviewer 2: ```suggestion
   * Review the information and deploy the Palette virtual cluster. The Palette virtual cluster will be provisioned within the next few minutes.
```
